src/index.js

```diff
@@ -202,6 +202,27 @@ function checkReturnForState(config, editorState, ev) {
   return newEditorState;
 }
 
+const unstickyInlineStyles = (character, editorState) => {
+  const selection = editorState.getSelection();
+  if (!selection.isCollapsed()) return editorState;
+
+  const startOffset = selection.getStartOffset();
+  const content = editorState.getCurrentContent();
+  const block = content.getBlockForKey(selection.getStartKey());
+  const entity = block.getEntityAt(startOffset);
+  if (entity !== null) return editorState;
```

Comment:
It'd be nice if you could also add a check for an inline entity

Reply:
How do I do that?

Reply:
hold on actually you're already doing that with `block.getEntityAt` - my bad :D